utils/create-news-verification-hashes/index.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * SCRIPT FOR NEWS FEED VERIFICATION HASH CREATION
+ *
+ * COMMANDS:
+ * 1. Fetch ALL files and create hashes:
+ *   yarn create-news-verification-hashes
+ *
+ * 2. Fetch file for specific environment and create hash:
+ *   NEWS_ENV=development yarn create-news-verification-hashes
+ */
+
+const lodash = require('lodash');
+const https = require('https');
```

Comment:
Unused `https` module

Reply:
@daniloprates my bad, I left that after script refactoring